packages/p2p/src/utils/string.js

```diff
@@ -8,3 +8,14 @@ export const toSentenceCase = string => {
 export const countDecimalPlaces = value => {
     return ((value.toString().split('.') || [])[1] || []).length;
 };
+
+export const generateHexColourFromNickname = nickname => {
+    const colours = ['#f43f83', 'var(--brand-secondary)', '#9ed178', 'var(--brand-orange)'];
+    const colour_hash = nickname.split().reduce((hash, char, idx) => {
+        /* eslint-disable no-bitwise */
+        const char_hash = nickname.charCodeAt(idx) + ((hash << 5) - hash);
```

Comment:
pls add a comment on what is going on here, at least about this calculation 🙏

Reply:
Done 👍